### ISSUE TITLE
Sub items?

### ISSUE THREAD

**Issue by srl295** [IBM-Bluemix/gp-angular-client#2]
In this usage:
`<label gp-translate="store"><input id='quick-store'></label> <br />`

… the nested `<input/>` element disappears once the translation appears.  Probably a `<span/>` could work here as a workaround.

**Comment by srl295** [comment]
So,
`    <label><span gp-translate="store"></span>:<input id='quick-store'></label> <br />`
works, but
`    <label><span gp-translate="store" />:<input id='quick-store'></label> <br />`
… does NOT. This may be a generic Angular or even HTML thing.

**Comment by DTownSMR** [comment]
Maybe related to what is talked about in this [issue](https://github.com/angular/angular.js/issues/1237) (says it has to do with the browsers parser)

**Comment by srl295** [comment]
@DTownSMR Yes - that issue makes sense. Not gp-angular or angular.js’s issue. Could be something to note in the docs (I'm sure I could figure out a PR here…)

Seems like the nested items use should also be at least documented, there's probably not an easy way around it.
